src/machines/authMachine.ts

```diff
@@ -162,15 +162,15 @@ async function getUser(input: { token?: string }) {
     }
   }
 
-  const userPromise = !isDesktop()
-    ? fetch(url, {
+  const userPromise = isDesktop()
+    ? getUserDesktop(token, VITE_KC_API_BASE_URL)
+    : fetch(url, {
         method: 'GET',
         credentials: 'include',
         headers,
       })
         .then((res) => res.json())
         .catch((err) => console.error('error from Browser getUser', err))
-    : getUserDesktop(input.token ?? '', VITE_KC_API_BASE_URL)
```

Comment:
All the logic for grabbing the token from cookies, localStorage or token.txt went into `const token = await getAndSyncStoredToken(input)` above, but then was ignored for this desktop request.